src/App.js

```diff
@@ -1,11 +1,10 @@
 import React from 'react';
+import ShowCombatCard from './components/ShowCombatCard'
 import './App.css';
 
 function App() {
   return (
-    <div className="App">
-      
-    </div>
+    
   );
```

Comment:
You shouldn't commit broken code, with this modification I cannot `npm start` your branch to see if your feature works.